I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 95, in script
    jump cont1
ScriptError: could not find label 'cont1'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 95, in script
    jump cont1
  File "C:\Users\Yusri Zafri\Desktop\renpy-8.0.3-sdk\renpy\ast.py", line 1974, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\Yusri Zafri\Desktop\renpy-8.0.3-sdk\renpy\script.py", line 927, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'cont1'.

Windows-10-10.0.19045 AMD64
Ren'Py 8.0.3.22090809
 Version 1.0
Mon Feb  6 18:06:09 2023
